src/app/calculator/furnaces/opening/opening.component.ts

```diff
@@ -0,0 +1,136 @@
+import { Component, ElementRef, HostListener, Input, OnInit, ViewChild } from '@angular/core';
+import { Subscription } from 'rxjs';
+import { SettingsDbService } from '../../../indexedDb/settings-db.service';
+import { OperatingHours } from '../../../shared/models/operations';
+import { OpeningLoss, OpeningLossOutput } from '../../../shared/models/phast/losses/openingLoss';
+import { Settings } from '../../../shared/models/settings';
+import { OpeningService } from './opening.service';
+
+@Component({
+  selector: 'app-opening',
+  templateUrl: './opening.component.html',
+  styleUrls: ['./opening.component.css']
+})
+export class OpeningComponent implements OnInit {
+  @Input()
+  settings: Settings;
+  @Input()
+  inTreasureHunt: boolean;
+  @Input()
+  operatingHours: OperatingHours;
+  
+  @ViewChild('leftPanelHeader', { static: false }) leftPanelHeader: ElementRef;
+  @ViewChild('contentContainer', { static: false }) contentContainer: ElementRef;
+  @HostListener('window:resize', ['$event'])
+  onResize(event) {
+    setTimeout(() => {
+      this.resizeTabs();
+    }, 100);
+  }
+  containerHeight: number;
+  isModalOpen: boolean;
+  modalSubscription: Subscription;
+
+  baselineData: Array<OpeningLoss>;
+  modificationData: Array<OpeningLoss>;
+  baselineDataSub: Subscription;
+  modificationDataSub: Subscription;
+  outputSubscription: Subscription;
```

Comment:
outputSubscription and output unused (i'm thinking this may be in the other furnace calculators as well)